watch/view.go

```diff
@@ -305,7 +305,7 @@ func (v *View) fetch(doneCh, successCh chan<- struct{}, errCh chan<- error) {
 	}
 }
 
-const minDelayBetweenUpdates = time.Millisecond * 100
+const minDelayBetweenUpdates = time.Millisecond * 1000
```

Comment:
This is still not really sufficient.  With this change, each HVS secret appearing in a template now only gets polled once per second.  But it wouldn't take very many templated secrets to start hitting API rate limits.